frontend/test/metabase-visual/dashboard/fullscreen.cy.spec.js

```diff
@@ -0,0 +1,67 @@
+import { restore } from "__support__/e2e/cypress";
+import { SAMPLE_DATASET } from "__support__/e2e/cypress_sample_dataset";
+
+const { PRODUCTS, PRODUCTS_ID } = SAMPLE_DATASET;
+
+const questionDetails = {
+  name: "15119",
```

Comment:
Please remove the reference to the issue 15119 to avoid the confusion in the future. Thanks

Reply:
👍 Done